database/staking.go

```diff
@@ -115,32 +115,42 @@ func (db BigDipperDb) SaveValidatorsData(validators []types.Validator) error {
 	validatorQuery := `INSERT INTO validator (consensus_address, consensus_pubkey) VALUES `
 	var validatorParams []interface{}
 
-	validatorInfoQuery := `INSERT INTO validator_info (consensus_address,operator_address,moniker,identity,website,security_contact, details) VALUES`
-
+	validatorInfoQuery := `INSERT INTO validator_info (consensus_address,operator_address,self_delegate_address,moniker,identity,website,security_contact, details) VALUES`
```

Comment:
Since the `self_delegate_address` references the `account` table, we should call `db.SaveUser` or whatever method is that allows us to save that address to make sure that it exists before referencing it during this insertion. Not doing so would later give an error that the account does not exist

Reply:
SelfDelegationAccQuery := `INSERT INTO account (address) VALUES `...
I am feeling already implement that saving into the account in line 122 in the same function. Do we need to insert it into balance table as well(like db.SaveAccount)?

Reply:
@HarleyAppleChoi Sorry, it's fine as you've done it here. I simply missed the query since it is declared below. Do you mind switching line 122 with 118 so that we first initialize the `INSERT INTO account` query and then the `INSERT INTO validator_info` query?